src/mod/endpoints/mod_sofia/mod_sofia.c

```diff
@@ -4904,6 +4904,11 @@ static switch_call_cause_t sofia_outgoing_channel(switch_core_session_t *session
 	int mod = 0;
 
 	*new_session = NULL;
+	
+	// Need to allow for override for outbound calls even when min-idle-cpu fails
+	if(mod_sofia_globals.limit_uuid_override_outbound) {
```

Comment:
Ideally a event/channel variable is useful in this case if we want to overwrite the default behavior. One useful case would be to a high profile customer that we don't want to reject their outgoing calls.

Reply:
If i'm reading your code then the default value for limit_uuid_override_outbound should be false or 0 so I would assume that this is the desire or default behavior where we reject outgoing call. So that's why I commented to add channel variable to enable that to true to force send the outgoing calls.

Reply:
It's defaulting to 0 simply due to the memset'ing.  That default should not be taken to indicate what is desired.  It's strictly a logical true or false for use with an if().  If in the long term this remains something that we can change on the fly, I'm not against pre-initializing it to 1, but it seemed silly to me since at this point in time the expectation is that the ENV variable will exist and be set to a value.